### PR TITLE
fix: Add support for builtin `char8_t` type

### PR DESCRIPTION
**Pull request by Swatinem**

### CODE REVIEW

File: src/ast.rs [hunk 1]
```diff
@@ -4044,6 +4044,7 @@ define_vocabulary! {
         DecimalFloat16   (b"Dh", "half"),
         Char32           (b"Di", "char32_t"),
         Char16           (b"Ds", "char16_t"),
+        Char8            (b"Du", "char8_t"),
```
Comment (khuey) [inline src/ast.rs:4047]: Please add this to the comment above too.

**Review by khuey — CHANGES_REQUESTED** [review]

**Comment by khuey** [comment]
Thanks!

File: src/ast.rs (hunk 1 above)
Comment (Swatinem) [reply src/ast.rs:4048]: done

**Comment by khuey** [comment]
I released 0.3.3 with this.

**Pull request merged**